### ISSUE TITLE
Flags should be added to NetCDF

### ISSUE THREAD

**Issue by j-harbin** [j-harbin/oceToNetCDF#62]

**Comment by j-harbin** [comment]
This will include adding a flag scheme

**Comment by j-harbin** [comment]
This has been added for CTD and RCM in of main. Note that when I added the `flagScheme` to the global attributes, I was initially going to do:

`    #odf <- oceSetMetadata(odf, name="flagScheme", value= list("name"= "argo", "mapping"=list("not_assessed"=0, "passed_all_tests"=1, "probably_good"=2, "probably_bad"=3, "bad"=4, "changed"=5, "not_used"=6, "not_used"=7, "estimated"=8, "missing"=9), "default"=c(0,3,4,9)))`

similar to an argo format, but it would not allow me to do a list. For now, I just put "Argo" which can be changed in the future.

**Comment by dankelley** [comment]
Re "it would not allow me to do a list", is that an oce bug?  If so, please raise an issue on the oce gh site.  Thanks.

**Comment by j-harbin** [comment]
Hi Dan- No I should have specified, `oceSetMethod` works fine, but `ndcf4` does not allow me to add a global attribute that is a list.

**Comment by dankelley** [comment]
Yeah, I like netcdf format but it is limited.  (Well, it's not unusually limited.  It is limited like ODF, limited like the tidyverse, etc. Most of these systems have trouble handling  nonlinear data types like lists, trees, etc.)

**Comment by j-harbin** [comment]
Reopening as they are not being added properly in adp type.

**Comment by j-harbin** [comment]
Done for standard=CF for ads type in commit 3ee859fd27e599c4f2d2fade7210dd5a9a829e79. I am now going to work on `standard='bodc'`
